common/app/views/fragments/email/stylesheets/front.scala.html

```diff
@@ -45,14 +45,17 @@
         text-decoration: none;
     }
     .facia-card__text {
-        padding: 0 10px 26px 10px !important;
+        padding: 0 10px 0px 10px !important;
```

Comment:
yes I will do - though there are quite a few places where this would be needed. Fortunately there is some hope - apparently gmail are rolling out updates which mean it will no longer strip `<style>` tags. Once this is the case we could get rid of style inlining, which would have lots of benefits, including potentially allowing us to remove `!important` from the base framework styles, and hence from everywhere else